backend/application/import_observations/parsers/prometheus_trivy/parser.py

```diff
@@ -0,0 +1,167 @@
+import json
+from typing import Optional
+
+import requests
+
+from application.core.models import Observation
+from application.core.types import Severity, Status
+from application.import_observations.models import Api_Configuration
+from application.import_observations.parsers.base_parser import (
+    BaseAPIParser,
+    BaseParser,
+)
+from application.import_observations.types import Parser_Type
+
+STATUS_MAPPING = {
```

Comment:
That are exactly the same values as in the Dependency Track parser. Are they coming from Trivy in Kubernetes as well?

Reply:
Do you mean the status mappings or the observation values?

Reply:
The status mappings. It implies Trivy on Kubernetes would use exactly the same status as Dependency Track, which seem unlikely to me.

Reply:
If Trivy outputs a vulneraility the status is alway open. There is no status value. I will adjust it later.